src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/service/kafka/KafkaProducerService.java

```diff
@@ -87,6 +89,14 @@ public void sendChunkedMessageAndTrack(String topic, String key, Collection<? ex
         }
     }
 
+    @Override
+    public void sendChunkedSpeakerDataAndTrack(String topic, MessageContext messageContext,
+                                               Collection<? extends InfoData> data) {
+        for (AbstractMessage message : ChunkedSpeakerDataResponse.createChunkedList(data, messageContext)) {
+            sendMessageAndTrack(topic, messageContext.getCorrelationId(), message);
```

Comment:
what if sending will fail in the middle of the loop? probably would be good to have some log message at least?

Reply:
We already have logging  in SendStatusCallback class.
So if anything fails in the middle of the loop, it will log from the KafkaProducerService side.

Also, it will fail the whole operation on the final consumer side as well. since we have mechanism that counts the  received messages